genome_kit/data_manager.py

```diff
@@ -185,7 +185,12 @@ def __init__(self, data_dir: str):
     def bucket(self):
         if not hasattr(self, "_bucket"):
             gcloud_client = storage.Client()
-            self._bucket = gcloud_client.bucket(_GCS_BUCKET, user_project=os.environ.get("GENOMEKIT_GCS_BILLING_PROJECT", None))
+            try:
+                self._bucket = gcloud_client.bucket(_GCS_BUCKET, user_project=os.environ.get("GENOMEKIT_GCS_BILLING_PROJECT", None))
+            except Exception as e:
+                # give the user a hint in case of permission errors
+                print(e)
```

Comment:
maybe it should go to stderr or a log? no opinion on this, but the follow suggestion needs an import of sys 
```suggestion
                print(e, file=sys.stderr)
```

Reply:
this needed to be done in both places and an import to sys?

Reply:
I'm a bit worried that the tests passed without this

Reply:
This is why:
https://github.com/deepgenomics/GenomeKit/pull/3/commits/ebba1c32f048c1c41f4bbf41638051e8f711d258